README.rst

```diff
@@ -37,6 +37,7 @@ DIRAC (Distributed Infrastructure with Remote Agent Control) INTERWARE is a soft
 
 DIRAC has been started by the `LHCb collaboration <https://lhcb.web.cern.ch/lhcb/>`_ who still maintains it. It is now used by several communities (AKA VO=Virtual Organizations) for their distributed computing workflows.
 
+DIRAC is written in python 2.7.13 and transitioning to python 3.
```

Comment:
Maybe it's a good opportunity to add that it currently only run fully on SLC6 ?

Reply:
This is for integration branch, by that time we'll support also CentOS7 by default.

Reply:
so maybe add the two ? I don't think we say it anywhere

Reply:
Done.